docs/CHANGELOG.md

```diff
@@ -6,27 +6,29 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+
+## 1.4.0 - 2021-12-13
```

Comment:
Date not correct

Reply:
Oops... I was distracted by the previous release's date being wrong, and then I forgot to change this one 🤦‍♂️